### ISSUE TITLE
Running process not being detected

### ISSUE THREAD

**Issue by sebasi** [hetrixtools/agent#49]
With commit e8293fa62a529b47cd49beab782b4e7f29660dee the changed expression to count running processes doesn't see one of my configured processes anymore.

```
# The process is in the ps output
root@host:/etc/hetrixtools# ps -ef | grep -w "DicomProcessor"
ultrast+  1410  1225  0 03:00 ?        00:01:43 DicomProcessor
root     29051 22167  0 13:18 pts/1    00:00:00 grep -w DicomProcessor

# old method
root@host:/etc/hetrixtools# ps -ef | grep -w "DicomProcessor" | grep -v "grep" | grep -v "/DicomProcessor/" | wc -l
1

# new method
root@host:/etc/hetrixtools# ps -ef | grep -E "[\/ ]DicomProcessor[^\/]" | grep -v "grep" | wc -l
0

# when removing the last character from the grep it works. Not sure if this is a word boundary/newline issue...
root@host:/etc/hetrixtools# ps -ef | grep -E "[\/ ]DicomProcesso[^\/]" | grep -v "grep" | wc -l
1
root@host:/etc/hetrixtools# ps -ef | grep -E "[\/ ]DicomProcessor[^\/]" | grep -v "grep" | wc -l
0
```

**Comment by hetrixtools** [comment]
Can you please check if this fixes your issue?
`ps -ef | grep -E "[\/ ]DicomProcessor([^\/]|$)" | grep -v "grep"`

**Comment by sebasi** [comment]
yes, that returns the expected line!
```
root@host:/etc/hetrixtools# ps -ef | grep -E "[\/ ]DicomProcessor([^\/]|$)" | grep -v "grep"
ultrast+  5943  5143  0 03:00 ?        00:02:07 DicomProcessor
```

**Comment by hetrixtools** [comment]
This has now been implemented in version `2.0.6` of our monitoring agent, with the following commit https://github.com/hetrixtools/agent/commit/a0c1b404fec7f8c9fc33c78cfc8b6c91c126d93e

Thank you for reporting the issue to us.
